docusaurus/docs/iOS/02-tutorials/01-video-calling.mdx

```diff
@@ -253,32 +280,35 @@ Now when you run the app, you'll see your local video in a floating video elemen
 
 Let's review the changes we made.
 
-[VideoRendererView](../../ui-components/video-renderer/) is one of our primary low-level components.
+We added the `changeTrackVisibility` in our app and we propagate its call to the other subviews we create. When this method is being called we are asking the Call object to make the participant's track visible or not visible. This is important when the view goes off-screen (e.g. when the user backgrounds the app) to reduce energy and data consumption.
```

Comment:
"e.g. when the user backgrounds the app" - better example would be when the user goes off-screen while scrolling through participants during a call. Pausing of tracks in the background is handled differently (listening to app lifecycle notifications over view lifecycle).